### ISSUE TITLE
New debug utils breaks Nativescript usage

### ISSUE THREAD

**Issue by neuspadrin** [500tech/ng2-mobx#12]
Love the simplicity of the plugin and it would be great if core features could remain usable in non-browser environments. The new debug options look useful, but for usage in native environments such as Nativescript with Angular the debug options break everything due to the assumption of the existence of "window".

I'm pretty sure a temp-fix would be to simply check for window's existence as well (with debugging unfortunately forced off for native use). For example below, wrapping the function being set on the window with a window check, and then adding "window &&" to the start of isDebugOn. Maybe future could add support for native debugging as well. 

```javascript
// function for turning debug on / off
if (window) {
  window['ng2MobxDebug'] = (value) => {
    if (value) window['localStorage']['ng2-mobx-debug'] = true;
    else delete window['localStorage']['ng2-mobx-debug'];
  }
}

function isDebugOn() {
  return window && window['localStorage'] && window['localStorage']['ng2-mobx-debug'];
}
```

For now just set myself to use 1.2.0 which looks to be last build before the debug options were added.

**Comment by adamkleingit** [comment]
Fixed in 1.2.3 - can you verify?

**Comment by adamkleingit** [comment]
Hey,
I'm closing the issue - if the problem persists please reopen. Thanks
